### ISSUE TITLE
doodle_2.13 artifact is empty

### ISSUE THREAD

**Issue by Iltotore** [creativescala/doodle#105]
Hello, the artifact `org.creativescala::doodle:0.9.25` is empty (only contains the manifest file). Is this intented ?

Here is the [Maven Search Page](https://search.maven.org/search?q=a:doodle_2.13)

**Comment by noelwelsh** [comment]
Thanks for letting me know. No idea why that would happen.

**Comment by SethTisue** [comment]
I think this might be normal? Because the .pom pulls in the JARs containing the actual code as dependencies?

**Comment by Iltotore** [comment]
> I think this might be normal? Because the .pom pulls in the JARs containing the actual code as dependencies?

The pom of this artifact only contains the scala library. This might be the cause of this issue.

**Comment by SethTisue** [comment]
indeed, that's what I see at https://repo1.maven.org/maven2/org/creativescala/doodle_2.13/0.9.25/doodle_2.13-0.9.25.pom

and it's a regression since 0.9.23; https://repo1.maven.org/maven2/org/creativescala/doodle_2.13/0.9.23/doodle_2.13-0.9.23.pom has the dependencies

@noelwelsh I looked at the diffs since then but the problem wasn't obvious to me. is it obvious to you? (if not, maybe we need to bisect to see which commit caused the problem?)

**Comment by Iltotore** [comment]
> > @noelwelsh I looked at the diffs since then but the problem wasn't obvious to me. is it obvious to you? (if not, maybe we need to bisect to see which commit caused the problem?)
> 
> I know approximately nothing about how publishing / POMs / Maven works, so it's safe to assume I don't have a clue what has caused the problem.

I don't use SBT but isnt [this line](https://github.com/creativescala/doodle/blob/e672704017d5c3924146a64cf8692d3f14f6b97d/publish.sbt#L31) the cause of this issue ?

**Comment by noelwelsh** [comment]
Thanks, I'll take a look when I get time.

**Comment by SethTisue** [comment]
I don't think `pomIncludeRepository` matters here.

I've submitted what I thiiiiink might be a fix at #108
